wonderland.js

```diff
@@ -2,16 +2,20 @@ import { tiny, defs } from "./external/common.js";
 import { Shape_From_File } from "./external/obj-file-demo.js";
 import { Custom_Movement_Controls } from "./custom-movement.js";
 
-const { Cube } = defs;
+const { Cube, Torus } = defs;
```

Comment:
Are you using `Torus` for anything?

Reply:
I did, and removed it and forgot to remove.